File: web-admin/app/shared/clubLogos.ts
export const CLUB_LOGO_FILES: Record<string, string> = {
  robotik: "robotik.png",
  tiyatro: "tiyatro.png",
  yazilim: "yazilim_kulubu.png",
  girisimcilik: "girisimcilik.png",
  fotografcilik: "fotografcilik.png",
};

export function clubLogoPath(logoKey?: string | null): string | null {
  const key = logoKey?.trim();
  if (!key) return null;
  const fileName = CLUB_LOGO_FILES[key];
  if (!fileName) return null;
  return `/clubs/${fileName}`;
}

File: mobile/lib/shared/club_logos.dart
const clubLogoFiles = <String, String>{
  'robotik': 'robotik.png',
  'tiyatro': 'tiyatro.png',
  'yazilim': 'yazilim_kulubu.png',
  'girisimcilik': 'girisimcilik.png',
  'fotografcilik': 'fotografcilik.png',
};

String? clubLogoAssetPath(String? logoKey) {
  final key = logoKey?.trim();
  if (key == null || key.isEmpty) return null;
  final fileName = clubLogoFiles[key];
  if (fileName == null) return null;
  return 'assets/clubs/$fileName';
}
